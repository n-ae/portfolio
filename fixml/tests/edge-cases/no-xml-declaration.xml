<root>
  <test>File without XML declaration</test>
  <duplicate>Same content</duplicate>
  <duplicate>Same content</duplicate>
  <whitespace>   Spaces   and   tabs   </whitespace>
</root>