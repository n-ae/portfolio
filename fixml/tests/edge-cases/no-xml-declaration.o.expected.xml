<root>
  <test>File without XML declaration</test>
  <duplicate>Same content</duplicate>
  <whitespace>   Spaces   and   tabs   </whitespace>
</root>